### PR TITLE
Create SJET-AOCODAF405V3 target

### PR DESCRIPTION
**Pull request by lida2003**
Add SJET-AOCODAF405V3.config

### CODE REVIEW

**Comment by nerdCopter** [comment]
will need sync to https://github.com/betaflight/config as well.

**Review by haslinghuis — CHANGES_REQUESTED** [review]

File: configs/default/SJET-AOCODAF405V3.config [hunk 1]
```diff
@@ -124,10 +124,6 @@ serial 3 8192 115200 57600 0 115200
 serial 4 1 115200 57600 0 115200
 
 # master
-set pinio_config = 129,129,1,1
-set pinio_box = 40,41,42,255
-set serialrx_provider = CRSF
-set gps_provider = UBLOX
```
Comment (haslinghuis) [inline configs/default/SJET-AOCODAF405V3.config]: Do not remove pinio settings - maybe you misread as reviews shows more lines then intended.

Comment (haslinghuis) [reply configs/default/SJET-AOCODAF405V3.config]: Only remove UBLOX here.

Comment (lida2003) [reply configs/default/SJET-AOCODAF405V3.config]: So just remove serial rx and gps, am i right?

Comment (haslinghuis) [reply configs/default/SJET-AOCODAF405V3.config]: serial can be useful for >= 4.3.2

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**